test/parallel/test-http-outgoing-proto.js

```diff
@@ -12,3 +12,63 @@ assert.strictEqual(
   typeof ClientRequest.prototype._implicitHeader, 'function');
 assert.strictEqual(
   typeof ServerResponse.prototype._implicitHeader, 'function');
+
+// validateHeader
+assert.throws(common.mustCall(() => {
+  const outgoingMessage = new OutgoingMessage();
+  outgoingMessage.setHeader();
+}), /^TypeError: Header name must be a valid HTTP Token \["undefined"\]$/);
+
+assert.throws(common.mustCall(() => {
+  const outgoingMessage = new OutgoingMessage();
```

Comment:
To make the test more efficient, why not create a single top level `outgoingMessage` instance, each of the tests can just reuse it